node/node.go

```diff
@@ -1154,10 +1157,15 @@ func (app *App) listenToUpdates(ctx context.Context) {
 				}
 				if len(update.Data.ActiveSet) > 0 {
 					app.hOracle.UpdateActiveSet(update.Data.Epoch, update.Data.ActiveSet)
+					app.proposalBuilder.UpdateActiveSet(update.Data.Epoch, update.Data.ActiveSet)
+
+					if err := app.fetcher.GetAtxs(ctx, update.Data.ActiveSet); err != nil {
```

Comment:
you can drop this part. i covered that in my change https://github.com/spacemeshos/go-spacemesh/pull/5377